soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -196,7 +200,10 @@ static int siwg917_nwp_init(void)
 {
 	sl_wifi_device_configuration_t network_config;
 	sl_status_t status;
-
+#ifdef CONFIG_PM
+	sl_wifi_performance_profile_t performance_profile = {.profile =
+								     DEEP_SLEEP_WITH_RAM_RETENTION};
+#endif
```

Comment:
The prefered style is:
```
	sl_wifi_performance_profile_t performance_profile = {
		.profile = DEEP_SLEEP_WITH_RAM_RETENTION
	};
```

Then, you can change `#ifdef CONFIG_PM` in `if (IS_ENABLED(CONFIG_PM))`.

@ArunmaniAlagarsamy2710, watch this change. It will conflict with the Power Save PR.